File: ts-sdk/src/sdk/models/shared/getbalanceresponse.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { BalanceWithAssets } from "./balancewithassets";


export class GetBalanceResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=data" })
  data: BalanceWithAssets;
}

File: python-sdk/src/formanceapi/models/shared/getbalanceresponse.py
import dataclasses
from ..shared import balancewithassets as shared_balancewithassets
from dataclasses_json import dataclass_json
from formanceapi import utils


@dataclass_json
@dataclasses.dataclass
class GetBalanceResponse:
    data: shared_balancewithassets.BalanceWithAssets = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    